runtime/drive_handler.go

```diff
@@ -259,6 +259,7 @@ func newStubDrive(
 	jail jailer,
 	isWritable bool,
 	rateLimiter *proto.FirecrackerRateLimiter,
+	cacheType *string,
```

Comment:
`cacheType` should be a `string` type, not a `*string` pointer.

Reply:
The intention is to pass whatever values client sends to Firecracker, and let firecracker handle/validate the actual string. Also, that means FC-CD should not make any assumption on the default cache strategy that Firecracker has.

I am thinking of keeping the existing `newStubDrive()` as is and just adding a new receiver method to enable changing the `driveMount.cacheType` of a stubDrive? Something like this:
```
func (sd *stubDrive) WithCacheType(cacheType string) {
	if cacheType != "" {
		sd.driveMount.CacheType = cacheType
	}
}
```

Reply:
That's perfect.  If `firecracker-containerd` doesn't even need to be aware of the cache type values and just pass them through, it will be less fragile to upstream changes.